### ISSUE TITLE
OBJLoader not working

### ISSUE THREAD

**Issue by Blunderchips** [OskarVeerhoek/YouTube-tutorials#7]
i cant seem to get seem to get textues .obj to work using the loadTexturedModel() and createTexturedDisplayList() methods in the OBJLoader class, is there any special you need to do to get them to render the.obj?﻿

**Comment by OskarVeerhoek** [comment]
Yes, have you enabled the 'triangulate faces' option in your modeling program? It's imperative that the vertex data contains triangles and not quadrilaterals or lines.
